src/Components/ElementsOfSite/Header/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Logo from '../../Image/logo.png';
+
+interface IHeaderProps {
+  Menu: React.ReactNode;
+  InputT: React.ReactNode;
+}
+
+const Header: React.FC<IHeaderProps> = ({ Menu, InputT }) => {
+  return (
+    <nav className="navbar navbar-toggleable-md navbar-light bg-white fixed-top mediumnavigation">
+      <button
+        className="navbar-toggler navbar-toggler-right"
+        type="button"
+        data-toggle="collapse"
+        data-target="#navbarsExampleDefault"
+        aria-controls="navbarsExampleDefault"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span className="navbar-toggler-icon"></span>
+      </button>
+      <div className="container">
+        <img src={Logo} alt="" />
+        <div className="collapse navbar-collapse" id="navbarsExampleDefault">
+          {Menu}
+          {InputT}
```

Comment:
Вот эти компоненты импортируй в начале файла. Не передавая через пропсы.

```suggestion
          <Menu />
          <InputT />
```